app.js

```diff
@@ -40,7 +40,6 @@ var makeHeaderRow = function() {
   var thEl = document.createElement('th');
   thEl.textContent = ' ';
   trEl.appendChild(thEl);
-  // cookieTable.appendChild(trEl);
 
```

Comment:
The change is on master though so totally understand if my pr gets marked down for the omission.